data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAttesterDuties.java

```diff
@@ -106,47 +159,37 @@ public PostAttesterDuties(final DataProvider dataProvider, final JsonProvider js
         @OpenApiResponse(status = RES_SERVICE_UNAVAILABLE, description = SERVICE_UNAVAILABLE)
       })
   @Override
-  public void handle(Context ctx) throws Exception {
+  public void handle(@NotNull Context ctx) throws Exception {
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(RestApiRequest request) throws JsonProcessingException {
     if (!validatorDataProvider.isStoreAvailable() || syncDataProvider.isSyncing()) {
-      ctx.status(SC_SERVICE_UNAVAILABLE);
+      AsyncApiResponse.respondWithError(
+          SC_SERVICE_UNAVAILABLE,
+          "Beacon node is currently syncing and not serving request on that endpoint");
```

Comment:
can just use a normal response, don't need to use asyncApiResponse outside of the async stuff...